packages/editor/src/lib/components/Shape.tsx

```diff
@@ -181,6 +181,7 @@ const CulledShape = function CulledShape<T extends TLShape>({ shapeId }: { shape
 		'set shape stuff',
 		() => {
 			const bounds = editor.getShapeGeometry(shapeId).bounds
+			if (!bounds) return
```

Comment:
Temp solution for [this](https://discord.com/channels/859816885297741824/993857643221372940/1222495756384997487). Will remove.